src/shared/components/data-table/components/main/components/table-header/components/table-header-cell/TableHeaderCell.tsx

```diff
@@ -0,0 +1,114 @@
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+
+import QuestionButton from 'src/shared/components/question-button/QuestionButton';
+
+import { TableContext } from 'src/shared/components/data-table/DataTable';
+import {
+  type IndividualColumn,
+  SortingDirection
+} from 'src/shared/components/data-table/dataTableTypes';
+
+import SortIcon from 'static/icons/icon_arrow.svg';
+
+import styles from '../../TableHeader.scss';
+
+const TableHeaderCell = (props: IndividualColumn) => {
+  const { title, helpText, isSortable, columnId, width, headerCellClassName } =
+    props;
+
+  const { dispatch, sortedColumn } = useContext(TableContext) || {};
+
+  const { columnId: sortedColumnId, sortedDirection } = sortedColumn || {};
+
+  if (!dispatch) {
+    return null;
+  }
+
+  let currentColumnSortingDirection = SortingDirection.NONE;
+  if (columnId === sortedColumnId && sortedDirection === SortingDirection.ASC) {
+    currentColumnSortingDirection = SortingDirection.ASC;
+  }
+
+  const onSort = () => {
+    let directionToSet = SortingDirection.NONE;
+    if (
+      columnId !== sortedColumnId ||
+      sortedDirection === SortingDirection.NONE
+    ) {
+      directionToSet = SortingDirection.DESC;
+    } else if (sortedDirection === SortingDirection.DESC) {
+      directionToSet = SortingDirection.ASC;
+    } else if (sortedDirection === SortingDirection.ASC) {
+      dispatch({
+        type: 'clear_sorted_column'
+      });
+      return;
+    }
+
+    dispatch({
+      type: 'set_sorted_column',
+      payload: {
+        columnId,
+        sortedDirection: directionToSet
+      }
+    });
+  };
+
+  const sortArrowClassNames = classNames(styles.sortArrow, {
+    [styles.sortArrowActive]: columnId === sortedColumnId,
+    [styles.sortArrowUp]: currentColumnSortingDirection === SortingDirection.ASC
+  });
+
+  const headerCellClassNames = classNames(styles.headerCell, {
+    [styles.headerCellSortable]: isSortable,
+    headerCellClassName
+  });
+
+  const sortArrow = (
+    <div
+      onClick={isSortable ? onSort : undefined}
+      className={styles.sortArrowContainer}
+    >
+      <SortIcon className={sortArrowClassNames} />
+    </div>
+  );
+
+  return (
+    <th className={headerCellClassNames} style={{ minWidth: width }}>
+      <div className={styles.headerCellContent}>
+        {isSortable && sortArrow}
+
+        <span
+          onClick={isSortable ? onSort : undefined}
+          className={styles.title}
+        >
+          {title}
+        </span>
```

Comment:
- Notice the styling
![image](https://user-images.githubusercontent.com/6834224/188396813-adef29fe-5c37-481f-8b4a-e4c0b07de8d3.png)
- Notice that the table is expected to be sorted by default, and that the default sort column is e-value

Reply:
That's done